pages/osx/valet.md

```diff
@@ -0,0 +1,24 @@
+# valet
+
+> A minimalist Laravel development environment that allows hosting sites via local tunnels on `http://<example>.test`.
```

Comment:
Looks like the first suggestion of @sbrl was undo, maybe it was the problem with the commit user

```suggestion
A Laravel development environment that allows hosting sites via local tunnels on `http://<example>.test`.
```

Reply:
good catch. fixed